src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationTables.kt

```diff
@@ -7,11 +7,28 @@ package com.openlattice.launchpad.configuration
  */
 class IntegrationTables {
     companion object {
+
+        /**
+         * SQL for appending v2 columns to existing table
+         */
+        const val UPGRADE_V2= """
+           ALTER TABLE integration_activity
```

Comment:
I generally prefer us to use constants for table/column names (mostly for purposes of finding usage when refactoring or making updates) even though it makes it a bit harder to read the query. 

(This seems like it should probably be a separate ticket though since all the SQL here is done like this, and it's not super pressing since these tables are only scoped to launchpad)